apps/www/_blog/2023-08-01-react-native-storage.mdx

```diff
@@ -86,15 +86,14 @@ npx create-expo-app@latest cloudApp --template tabs@49
 
 # Install dependencies
 npm i @supabase/supabase-js
-npm i react-native-url-polyfill base64-arraybuffer react-native-loading-spinner-overlay
+npm i react-native-url-polyfill base64-arraybuffer react-native-loading-spinner-overlay @react-native-async-storage/async-storage
 
 # Install Expo packages
-npx expo install expo-secure-store
```

Comment:
Some of the blog below still refers to `secure-store`

line 149

Reply:
<img width="799" alt="Screenshot 2023-09-26 at 3 32 33 PM" src="https://github.com/supabase/supabase/assets/8291514/defc45b7-b1e2-4b37-a494-787307839883">

sorry, couldn't direct comment in PR. Should be able to find it.

Reply:
Nice catch, thank you!